### ISSUE TITLE
Instance Normalization not implemented right

### ISSUE THREAD

**Issue by StefanoD** [farizrahman4u/keras-contrib#126]
See [this](https://github.com/tensorflow/tensorflow/issues/11690#issuecomment-318187648) post.

**Comment by the-moliver** [comment]
See https://github.com/tensorflow/tensorflow/issues/11690#issuecomment-318373681

**Comment by StefanoD** [comment]
Ok, then I close this issue.
